pwiz_tools/Skyline/Model/TransitionGroupDocNode.cs

```diff
@@ -137,12 +159,28 @@ protected override IList<DocNode> OrderedChildren(IList<DocNode> children)
         /// Deep copy for use in small molecule user editing - we have to deep copy entire branches for changes to IDs
         /// </summary>
         /// <returns> Copy of transition group and its transitions, all with new IDs and backpointers</returns>
-        public TransitionGroupDocNode UpdateSmallMoleculeTransitionGroup(Peptide parentNew, TransitionGroup groupNew, SrmSettings settings)
+        public TransitionGroupDocNode DeepCopyTransitionGroup(Peptide parentNew, TransitionGroup groupNew,

```

Comment:
There is already a method "ChangeTransitionGroupId" which I think does the same thing as this.